src/components/ProjectDescriptionStepper/ProjectDescriptionStepper.tsx

```diff
@@ -34,6 +35,7 @@ const ProjectDescriptionStepper = ({
   useProjectDescriptionStepper = _useProjectDescriptionStepper,
   ...props
 }: ProjectDescriptionStepperProps) => {
+  const { t } = useTranslation()
   const ref = useRef<HTMLDivElement>(null)
   const [activeStep, setActiveStep] = React.useState(0)
```

Comment:
Maybe typisierung noch.
```suggestion
  const [activeStep, setActiveStep] = React.useState<number>(0)
```

Reply:
React.useState(0) -> useState(0)